From 2cac94185824aa7df07ec48a2872f3d26d517a6d Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Tue, 28 Nov 2023 10:23:42 +0800
Subject: [PATCH] launch: allow pipewire-pulse can be started by root.

revert commit 8942f6b40 ("launch: avoid autostarting pipewire-pulse
systemd units for root")

Upstream-Status: Inappropriate [i.MX specific]
Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
---
 src/daemon/systemd/user/pipewire-pulse.service.in | 1 -
 src/daemon/systemd/user/pipewire-pulse.socket     | 1 -
 2 files changed, 2 deletions(-)

diff --git a/src/daemon/systemd/user/pipewire-pulse.service.in b/src/daemon/systemd/user/pipewire-pulse.service.in
index 73d22e532..da7728ee3 100644
--- a/src/daemon/systemd/user/pipewire-pulse.service.in
+++ b/src/daemon/systemd/user/pipewire-pulse.service.in
@@ -14,7 +14,6 @@ Description=PipeWire PulseAudio
 # After=pipewire-pulse.socket is not needed, as it is already implicit in the
 # socket-service relationship, see systemd.socket(5).
 Requires=pipewire-pulse.socket
-ConditionUser=!root
 Wants=pipewire.service pipewire-session-manager.service
 After=pipewire.service pipewire-session-manager.service
 Conflicts=pulseaudio.service
diff --git a/src/daemon/systemd/user/pipewire-pulse.socket b/src/daemon/systemd/user/pipewire-pulse.socket
index 1ae5edafb..d27fb0e26 100644
--- a/src/daemon/systemd/user/pipewire-pulse.socket
+++ b/src/daemon/systemd/user/pipewire-pulse.socket
@@ -1,6 +1,5 @@
 [Unit]
 Description=PipeWire PulseAudio
-ConditionUser=!root
 Conflicts=pulseaudio.socket
 
 [Socket]
-- 
2.34.1

